packages/apidom-ns-asyncapi-3/src/elements/nces/OperationReplyMessage.ts

```diff
@@ -0,0 +1,15 @@
+import { ArrayElement, Attributes, Meta } from '@swagger-api/apidom-core';
+
+/**
+ * @public
+ */
+class OperationReplyMessage extends ArrayElement {
```

Comment:
Here and possibly in other places
```suggestion
class OperationReplyMessages extends ArrayElement {
```